.gitignore

```diff
@@ -273,4 +273,8 @@ $RECYCLE.BIN/
 # Logs
 /logs/*
 
+database/usersScript.sql
+
+database/usersScript.sql
+
```

Comment:
Remove these, the ignore for database/usersScript.sql is at the top of the .gitignore already.